frontend/src/components/submission-comments/add-comment.tsx

```diff
@@ -62,14 +63,15 @@ export const AddComment: React.SFC<Props> = ({ submissionId }) => {
   return (
     <Box as="form" onSubmit={onSubmit} sx={{ mb: 4 }}>
       <InputWrapper sx={{ mb: 2 }} errors={getErrors("validationText")}>
-        <Textarea
+        <LimitedTextarea
```

Comment:
we don't use Textarea anymore, so we can remove the import

Reply:
@estyxx this is still not solved. You can see above there is the eslint warning about the import